src/package/Dependencies.js

```diff
@@ -85,7 +85,8 @@ class Dependencies extends AbstractService {
       console.log(chalk.white(await this.run(this.plugin.settings.customInstallationCommand)));
     } else {
       const commands = this.plugin.runtimes.getCommands();
-      console.log(chalk.white(await this.run(commands[this.plugin.settings.packageManager])));
+      const {packageManagerExtraArgs, packageManager} = this.plugin.settings;
+      console.log(chalk.white(await this.run(`${packageManager} ${packageManagerExtraArgs}`)));
```

Comment:
Please change this line to      
```
console.log(chalk.white(await this.run(`${packageManager} install ${packageManagerExtraArgs}`)));
```
this way it will not break for people who don't provide any extra commands

I checked and all 4 package managers (yarn, npm, pip and bundle) use `install`

You may also want to add 
1. `--production --only=prod` for npm
2. `--production` for yarn
2. `-r ${this.default.dependenciesPath} -t` for pip
3. `--gemfile=${this.default.dependenciesPath} --path=./` for bundle
but I will leave that up to you
I can make a pr to have packageManagerDefaultArgs but I wouldn't know how to handle the yarn part since I don't really understand how that works

Reply:
This is what I used in my serverless.yml `packageManagerExtraArgs: 'i --ignore-scripts --production --only=prod'`

Reply:
Actually, this is an issue. I've fixed that in version `2.5.2-beta.2`. You should be able to do something like:

```
custom:
  serverless-layers:
    packageManagerExtraArgs: '--ignore-scripts --option a --option b...'
```

Reply:
Not necessary to specify `install`